t/e2e/jail/jail_rpc_test.go

```diff
@@ -94,6 +97,9 @@ func (s *JailRPCTestSuite) TestIsConnected() {
 
 // regression test: eth_getTransactionReceipt with invalid transaction hash should return "error":{"code":-32000,"message":"unknown transaction"}
 func (s *JailRPCTestSuite) TestRegressionGetTransactionReceipt() {
+	if SkipTransactionTest() {
```

Comment:
It's just checking if the proper error was returned. I think it's safe to run on mainnet.